tasks/tasks.js

```diff
@@ -6,37 +8,44 @@ const { MessageEmbed } = require('discord.js'),
 
 /*
 Cron job format =  '* * * * * *'
-Sec(0-59), min(0-59), hour(0-23), day of month(1-31), month(1-12), day of week(0-6 starting with sunday)
+Sec(0-59), min(0-59), hour(0-23), day of month(1-31), month(0-11), day of week(0-6 starting with sunday)
 */
 
+// 1 11 * * 1
 const weeklyLeaderboardResults = (client) => new CronJob('1 11 * * 1', async function() {
-    try {
-        const chartUrl = await getLeaderboardGraph();
+    const count = await sequelize.query('SELECT COUNT(*) FROM Users;');
+    if (count[0][0]['COUNT(*)'] > 0) {
```

Comment:
Maybe in the future we can see if there's a cleaner way to extract the `count` from this query response